### PR TITLE
Fix Composer issues

### PR DESCRIPTION
**Pull request by szepeviktor**
Please see the diff 👀

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,21 +16,21 @@
...
         "voku/simple-cache": "~4.0",
         "nikic/php-parser": "~4.4",
         "ondrejmirtes/better-reflection": "~4.3",
-        "vimeo/psalm": "~3.11",
         "symfony/console": "~3.4 || ~4.1 || ~5.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~6.0 || ~7.0"
+        "phpunit/phpunit": "~6.0 || ~7.0",
+        "vimeo/psalm": "~3.11"
```
Comment (voku) [inline composer.json:33]: We use psalm in the code, so we can't move it into the dev section.

Comment (szepeviktor) [reply composer.json:33]: Oh noooo!

`$this->typeFromPhpDocPslam = (string) @\Psalm\Type::parseString($this->typeFromPhpDoc);`

Comment (szepeviktor) [reply composer.json:33]: Does it make the PHAR much bigger?

Comment (szepeviktor) [reply composer.json:33]: 19 MB vs. 12 MB

Comment (szepeviktor) [reply composer.json]: Done.

Comment (voku) [reply composer.json]: Maybe we can use https://github.com/phpstan/phpdoc-parser here, instead of the full psalm package?

Comment (szepeviktor) [reply composer.json]: Yes, it understand some psalm stuff

https://github.com/phpstan/phpdoc-parser/search?q=psalm&unscoped_q=psalm

I am not able to execute this swap.

Comment (voku) [reply composer.json]: I will try it, when I have time. 12 MB sounds better than 19 MB. I have to make some food for my little daughter now. :) Thanks for your testing and the fixes! :+1:

Comment (voku) [reply composer.json]: https://github.com/voku/Simple-PHP-Code-Parser/commit/217dd6251ef2c6e63fd6eda8f6e2eefecd1b93a7 ⇉ 13 MB for the phar file :)

**Pull request merged**